src/main/java/com/stripe/model/EventDataClassLookup.java

```diff
@@ -7,8 +7,10 @@
 /**
  * Event data class look up used in event deserialization. The key to look up is `object` string of
  * the model.
+ *
+ * <p>For internal use by Stripe SDK only.
  */
-final class EventDataClassLookup {
+public final class EventDataClassLookup {
```

Comment:
Why did we have to make this public? Is it so that we can access it from StripeObject using the fully qualified name which was needed to differentiate between the v1 and v2 EventDataClassLookup? If that was the case that how about renaming this class to `V1EventDataClassLookup` and the other one to `V2EventDataClassLookup` and not making them pubic?

Reply:
Yes, I thought of it but for getting this out faster I picked this approach.